dataflow/gpu-examples/tensorflow-landsat/main.py

```diff
@@ -66,6 +66,9 @@
 DEFAULT_MAX_BAND_VALUE = 12000.0
 DEFAULT_GAMMA = 0.5
 
+DEFAULT_GPU_TYPE = "nvidia-tesla-t4"
```

Comment:
Link to docs which show different types.

Reply:
Thanks, I added a comment and a link to https://cloud.google.com/dataflow/docs/resources/locations#gpu_availability

I also added a similar note in the README when running the pipeline and passing the GPU type.